digitalocean/kubernetes/kubernetes.go

```diff
@@ -442,6 +442,10 @@ func FilterTags(tags []string) []string {
 
 func expandCAConfigOpts(config []interface{}) *godo.KubernetesClusterAutoscalerConfiguration {
 	caConfig := &godo.KubernetesClusterAutoscalerConfiguration{}
+	if len(config) == 0 || config[0] == nil {
```

Comment:
One more drive-by fix: cluster with `cluster_autoscaler_configuration {}` (empty but non-nil) used to cause a panic before this fix when trying to type-assert `nil` as `configMap := config[0].(map[string]interface{})` in the code below.